style.css

```diff
@@ -10,117 +10,119 @@ body {
 #navbar {
   position: fixed;
   min-width: 290px;
-  top: 0px;
-  left: 0px;
+  top: 0;
+  left: 0;
   width: 300px;
   height: 100%;
   border-right: solid;
   border-color: rgba(0, 22, 22, 0.4);
 }
+
 header {
-    color: black;
-    margin: 10px;
-    text-align: center;
-    font-size: 1.1em;
-    font-weight: thin;
-  }
-  
-  #main-doc header {
-    text-align: left;
-    margin: 0px;
-  }
-  
+  color: black;
+  margin: 10px;
+  text-align: center;
+  font-size: 1.1em;
+  font-weight: thin;
+}
+
+#main-doc header {
+  text-align: left;
+  margin: 0;
+}
+
+#navbar ul {
+  height: 88%;
+  padding: 0;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+section li {
+  margin: 15px 0 0 20px;
+}
+
+#navbar li {
+  color: #4d4e53;
+  border-top: 1px solid;
+  list-style: none;
+  position: relative;
+  width: 100%;
+}
+
+#navbar a {
+  display: block;
+  padding: 10px 30px;
+  color: #485386;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+#main-doc {
+  position: absolute;
+  margin-left: 310px;
+  padding: 20px;
+  margin-bottom: 110px;
+}
+
+section article {
+  color: #08144d;

```

Comment:
you can create a class and add it to all element that need to be repeated

Reply:
- [ ] Just as I suggested above, try to make your review comments a suggestion instead of an order. Also, try to give reasons for your suggested actions or changes. :+1: :small_airplane: 
- [OPTIONAL] I'll suggest you also try to use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.